circuits/cpp/src/aztec3/circuits/rollup/base/.test.cpp

```diff
@@ -129,8 +129,11 @@ class base_rollup_tests : public ::testing::Test {
         // uint8_t const* proof_data;
         // size_t proof_data_size;
         uint8_t const* public_inputs_buf;
+        size_t public_inputs_size;
```

Comment:
do we want to initialise this to anything before passing it in or declare it as a pointer instead if its behaving like one?

Reply:
Actually this is a nice way to do it. This way we don't have to do any manual memory management.